tutorials/scripting/gdscript/static_typing.rst

```diff
@@ -396,6 +396,53 @@ if you prefer a more readable and reliable, but more verbose syntax.
 ``UNSAFE_*`` warnings make unsafe operations more noticeable, than unsafe lines.
 Currently, ``UNSAFE_*`` warnings do not cover all cases that unsafe lines cover.
 
+Common unsafe operations and their safe counterparts
+----------------------------------------------------
+
+Warning: ``UNSAFE_CAST``
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the example below, we would like an object entering our collision area to take
+damage if it has a health variable that contains a ``Health`` object. The physics
+system passes us a ``Node2D`` object, and after we check whether that object has
+the health variable, we cast the variable to a ``Health`` object and call
+``take_damage()``. However, this gives an ``UNSAFE_CAST`` warning because
+``body.health`` is of an unknown type::
+
+    func _on_body_entered(body: Node2D) -> void:
+        if "health" in body:
+            (body.health as Health).take_damage(damage)
+
+This operation can be made type safe using the ``Object.get()`` method.
+``body.get("health")`` will return ``null`` if the property doesn't exist or if
+it is of another type::
```

Comment:
```suggestion
``body.get("health")`` will silently return ``null`` if the property doesn't exist::
```

When calling `get()`, you cannot specify what type you are expecting. This is not a special syntax, but a regular method ([`Object.get()`](https://docs.godotengine.org/en/stable/classes/class_object.html#class-object-method-get)).